shopelectro/selenium/pages/category.py

```diff
@@ -18,11 +20,37 @@ def __init__(self, driver, slug):
     def path(self):
         return reverse('category', args=(self.slug,))
 
-    def product_cards(self) -> typing.List[CatalogCard]:
-        raise NotImplementedError
+    def product_cards(self) -> typing.List[elements.CatalogCard]:
+        products_count = len(self.driver.find_elements(
+            By.CLASS_NAME, 'product-card'
+        ))
+        return [elements.CatalogCard(self.driver, index=i) for i in range(products_count)]
+
+    def find_card(self, id_: int) -> elements.CatalogCard:
+        return elements.CatalogCard(self.driver, id_=id_)
 
-    def add_to_cart(self, products: typing.List[CatalogCard] = None):
-        default = [CatalogCard(self.driver, i) for i in range(1, 7)]
+    def load_more(self):
+        old_len = len(self.product_cards())
+        load_more_locator = (By.ID, 'btn-load-products')
```

Comment:
```suggestion
        locator = (By.ID, 'btn-load-products')
```

we are already in `load_more` method. Up to you